docs/library/network.WLAN.rst

```diff
@@ -85,7 +85,10 @@ Methods
         * ``STAT_GOT_IP`` -- connection successful.
 
     When called with one argument *param* should be a string naming the status
-    parameter to retrieve.  Supported parameters in WiFI STA mode are: ``'rssi'``.
+    parameter to retrieve.  Supported parameters in WiFi STA mode are: ``'rssi'``.
+    In WiFi AP mode on ESP8266 the ``'stations'`` parameter is also supported,
```

Comment:
This needs to be updated to state that more ports support `stations`.  Maybe something like "some network interfaces in AP mode support the `stations` parameter which returns a list of connected STAs ...".

Reply:
I've expanded the documentation to add the list of ports involved, also for the `rssi` parameter in STA mode - as I split the text block into separate STA/AP paragraphs.